internal/enterprisehooks/agent_version_other.go

```
@@ -0,0 +1,36 @@
+//go:build !windows
+
+package enterprisehooks
+
+// discoverWindowsAgentVersion is a Windows-only feature — this stub
+// exists so the package compiles cross-platform. The macOS
+// enumerator (packaging/macos/lib/render-targets.sh) already covers
+// the darwin side via `discover_agent_version`; nothing about this
+// Go function is invoked on non-Windows OSes. The stub returns
+// empty so any accidental non-Windows caller would drop the row,
+// which is the safe default.
+func discoverWindowsAgentVersion(profileHome, connectorName string) string {
```

golangci-lint (2.12.2)

[error] func discoverWindowsAgentVersion is unused

(unused)